charts/dapr/charts/dapr_sidecar_injector/templates/dapr_sidecar_injector_deployment.yaml

```diff
@@ -72,7 +72,6 @@ spec:
         imagePullPolicy: {{ .Values.global.imagePullPolicy }}
 {{- if eq .Values.global.daprControlPlaneOs "linux" }}
         securityContext:
-          runAsUser: 1000
```

Comment:
Can you keep it when runAsNonRoot is false? Maybe surround it with an "if" condition.

Reply:
done.